src/main/java/session_17_equals_hashcode_generics/challenge/challenge_6/LightColor.java

```diff
@@ -0,0 +1,29 @@
+package session_17_equals_hashcode_generics.challenge.challenge_6;
+
+public enum LightColor {
+    RED("Stop", 30),
+    YELLOW("Caution", 5),
+    GREEN("Go", 45);
+
+
```

Comment:
Please consider removing an empty row here. We have 2 of them and we need just one.